src/main/java/io/fabric8/elasticsearch/util/RequestUtils.java

```diff
@@ -78,16 +78,16 @@ public String getUser(RestRequest request) {
     }
     
     public String getBearerToken(RestRequest request) {
-        String token = "";
-        if (request.header(AUTHORIZATION_HEADER) != null) {
-            final String[] auth = StringUtils.defaultIfEmpty(request.header(AUTHORIZATION_HEADER), "").split(" ");
-            if (auth.length >= 2 && "Bearer".equals(auth[0])) {
-                token = auth[1];
+        String token = request.header(X_FORWARDED_ACCESS_TOKEN);
```

Comment:
does `request.header` do case insensitive comparison?  If not, do you need to somehow make all of the request headers lower case before checking?  Same with `request.header(AUTHORIZATION_HEADER)` below.

Reply:
it is case insensitive to the value per the documentation.